pennylane/grouping/group_observables.py

```diff
@@ -49,16 +49,13 @@ class PauliGroupingStrategy:  # pylint: disable=too-many-instance-attributes
     Args:
         observables (list[Observable]): a list of Pauli words to be partitioned according to a
             grouping strategy
-
-    Keyword Args:
```

Comment:
`Keyword Args:` section was removed such that named arguments with defaults go in `Args:` (recurring change)